src/main/java/uk/gov/hmcts/reform/sscs/service/ccdupdate/CcdStateUpdateService.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.hmcts.reform.sscs.service.ccdupdate;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.State;
+import uk.gov.hmcts.reform.sscs.exception.UpdateCaseException;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingGetResponse;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.ListingCaseStatus;
+import uk.gov.hmcts.reform.sscs.reference.data.mappings.CancellationReason;
+
+import javax.validation.Valid;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.State.DORMANT_APPEAL_STATE;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.State.HEARING;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.State.READY_TO_LIST;
+import static uk.gov.hmcts.reform.sscs.model.single.hearing.ListingStatus.FIXED;
+
+
+@Slf4j
+@Service
+public class CcdStateUpdateService {
+
+    private static final String CANCELLED = "Cancelled";
+
+    public void updateListed(HearingGetResponse hearingResponse, @Valid SscsCaseData sscsCaseData)
+            throws UpdateCaseException {
+
+        if (!isHearingListingStatusFixed(hearingResponse)) {
```

Comment:
optional: might be a pain but I would invert this method so the logic follows the flow so make it `hearingListingStatusIsNotFixed` and change the necessary logic.